packages/sitelen-pona/0.1.0/_manifest.yml

```diff
@@ -0,0 +1,5 @@
+name: "sitelen-pona"
+title: "o sitelen pona!"
+description: A package for writing UCSUR sitelen pona with Espanso!
+version: 0.1.0
+author: niko
```

Comment:
This needs some `tags:`. See [espanso hub](https://hub.espanso.org/search) for examples.